cmd/stellar-rpc/internal/integrationtest/infrastructure/test.go

```diff
@@ -70,15 +72,22 @@ type TestConfig struct {
 	SQLitePath string
 	OnlyRPC    *TestOnlyRPCConfig
 	// Do not mark the test as running in parallel
-	NoParallel bool
+	NoParallel                bool
+	EnableCoreHTTPQueryServer bool
 }
 
 type TestCorePorts struct {
 	CorePort        uint16
 	CoreHTTPPort    uint16
 	CoreArchivePort uint16
-	// This only needs to be an unconflicting port
-	captiveCorePort uint16
+
+	// These only need to be unconflicting ports
+	captiveCorePort          uint16
```

Comment:
@2opremio @Shaptic do you still need this `captiveCorePort` field given that we have introduced `captiveCoreHTTPPort`? It seems like `captiveCorePort` and `captiveCoreHTTPPort` are both trying to configure the same port